apps/docs/src/components/landing/FeatureSection.tsx

```diff
@@ -0,0 +1,90 @@
+import { faArrowRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useState, useEffect } from 'react';
+interface Feature {
+  title: string;
+  icon: any; // You can specify the correct type for FontAwesomeIcon
+  text: string;
+  hoverImage: string;
+  shadow?: boolean;
+}
+
+interface FeatureSectionProps {
+  title: string;
+  description: string;
+  features: Feature[];
+}
+
+function FeatureSection({ title, description, features }: FeatureSectionProps) { 
+  const [hoveredImage, setHoveredImage] = useState(features[0]?.hoverImage || '');
```

Comment:
Could we somehow implement the state logic in a way that the stage also persists when one leaves the corresponding component with the mouse? Currently it always seems to jump back to the initial vaule

Reply:
Possibly just removing the handleMouseLeave function could already be sufficient?

Reply:
need to look at this removing, the handlemouseleave function doesn't seem to be sufficient.

Reply:
done :)